packages/palette_generator/README.md

```diff
@@ -13,7 +14,7 @@ To use this package, add `palette_generator` as a
 ## Example
 
 Import the library via
-
+<?code-excerpt "main.dart (Import)"?>
 ```dart
 import 'package:palette_generator/palette_generator.dart';
```

Comment:
This should just be removed; these kinds of instructions predate the `install` tab in pub.dev, which already includes this information, and we've been incrementally removing them from all of our packages.

Reply:
Done. I have removed it.